src/transformers/models/auto/configuration_auto.py

```diff
@@ -474,6 +475,7 @@
         ("ul2", "UL2"),
         ("unispeech", "UniSpeech"),
         ("unispeech-sat", "UniSpeechSat"),
+        ("upernet", "UPerNet"),
```

Comment:
There should also be a line with the archive constant.

Reply:
Could you clarify what you mean by "archive constant"?

Reply:
[This one](https://github.com/huggingface/transformers/blob/1543cee7c8c95ef47f832b1f37625ba2923c4994/src/transformers/models/auto/configuration_auto.py#L186)

Reply:
Thanks, fixed